### PR TITLE
Add channels methods.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-963.

### CODE REVIEW

File: data/misc.go [hunk 1]
```diff
@@ -72,6 +72,16 @@ func HexToAddress(addr string) (common.Address, error) {
 	return common.BytesToAddress(addrBytes), err
 }
 
+// EthAddrFromHex returns ethereum's address on string format from hex encoded
+// string if the address is not valid, it returns an empty ethereum's address.
+func EthAddrFromHex(addr string) string {
+	ethAddr, err := HexToAddress(addr)
+	if err != nil {
+		ethAddr = common.Address{}
```
Comment (furkhat) [inline data/misc.go]: warning or error log seems good here.

File: ui/export_test.go [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+	"github.com/privatix/dappctrl/data"
+	"github.com/privatix/dappctrl/job"
+	"github.com/privatix/dappctrl/proc"
+)
 
 func (h *Handler) SetMockQueue(q job.Queue) {
 	h.queue = q
 }
+
+func (h *Handler) SetRole(role string) {
```
Comment (furkhat) [inline ui/export_test.go]: Can we add `Mock` to the name of the function? eg `SetMockRole`

File: ui/misc.go [hunk 3]
```diff
@@ -1,10 +1,11 @@
 package ui
 
 import (
-	"database/sql"
-	"strconv"
+	"fmt"
 
+	"database/sql"
 	"gopkg.in/reform.v1"
+	"strconv"
```
Comment (furkhat) [inline ui/misc.go]: they belong to the first (built-in packages) group.

File: main.go [hunk 4]
```diff
@@ -293,8 +294,8 @@ func main() {
 	go func() {
 		fatal <- uiSrv.ListenAndServe()
 	}()
-
-	uiSrv2, err := createUIServer(conf.UI, logger, db, queue, pwdStorage)
+	uiSrv2, err := createUIServer(conf.UI, logger, db, queue, pwdStorage,
+		bool(conf.Role == data.RoleAgent), pr)
```
Comment (ababo) [inline main.go]: Why do we need to explicitly convert to `bool`?

**Pull request merged**